src/options/help.rs

```diff
@@ -46,6 +46,7 @@ LONG VIEW OPTIONS
   -H, --links          list each file's number of hard links
   -i, --inode          list each file's inode number
   -m, --modified       use the modified timestamp field
+  -n, --numeric-uid-gid list numeric user and group IDs
```

Comment:
There should two spaces between the option name and description:

```suggestion
  -n, --numeric-uid-gid  list numeric user and group IDs
```

This will be displayed with `exa --help`. Please align the descriptions to the same column.

Reply:
`--numeric-uid-gid` is too long to make the description align with other descriptions. Adding more spaces makes the alignment off even more. I will add a space to other options to make their descriptions align the description of `-n`.